tools/opt/opt.cpp

```diff
@@ -338,6 +339,11 @@ Options (in lexicographical order):
   --strip-reflect
                Remove all reflection information.  For now, this covers
                reflection information defined by SPV_GOOGLE_hlsl_functionality1.
+  --target-env=<env>
+               Set the target environment. Without this flag the target
+               enviroment defaults to spv1.3.
+               <env> must be one of vulkan1.0, vulkan1.1, opencl2.2, spv1.0,
+               spv1.1, spv1.2, spv1.3, or webgpu0.
```

Comment:
State the default value here?

Reply:
Ah, never mind.  This is only done if the flag is passed, so the default is set elsewhere